--- data/org.pantheon.scratch.appdata.xml.in
+++ data/org.pantheon.scratch.appdata.xml.in
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- Copyright 2015 elementary, LLC <contact@elementary.io> -->
 <component type="desktop">
-  <id>scratch-text-editor.desktop</id>
+  <id>org.pantheon.scratch.desktop</id>
   <metadata_license>CC0-1/0</metadata_license>
   <project_group>elementary</project_group>
   <project_license>GPL-3.0+</project_license>

